api/src/main/java/marquez/db/ColumnLineageDao.java

```diff
@@ -88,4 +95,59 @@ void doUpsertColumnLineageRow(
               },
               value = "values")
           List<ColumnLineageRow> rows);
+
+  @SqlQuery(
```

Comment:
Most important piece of the PR: recursive query to extract column-lineage graph. 
Only `column_lineage` table is used and joined to obtained graph nodes. 
Other tables are only used to enrich found nodes.